circle.yml

```diff
@@ -242,20 +255,20 @@ commands:
               echo "export POM_VERSION_NAME=$POM_VERSION_NAME" >> $BASH_ENV
             fi
 
-  run-firebase-instrumentation:
+  run-internal-firebase-instrumentation:
     parameters:
       module_wrapper:
-        description: module wrapper
+        description: tests wrapper
         type: string
       module_target:
-        description: module target
+        description: SDK module target
         type: string
       variant:
         description: debug or release
         type: string
     steps:
       - run:
-          name: Run instrumentation tests on Firebase
+          name: Run instrumentation tests on intenral SDK classes on Firebase
```

Comment:
```suggestion
          name: Run instrumentation tests on internal SDK classes on Firebase
```

Reply:
Adjust next time? :)

Don't want to rerun the whole suite for this typo.

Reply:
https://github.com/mapbox/mapbox-navigation-android/pull/3895/commits/f036f1115fd33a48d7c22a4e8693b2a96251ba73